recipes/benchmarks/fmbench/README.md

```diff
@@ -0,0 +1,117 @@
+
+## [`FMBench`](https://github.com/aws-samples/foundation-model-benchmarking-tool/tree/main): an open-source Python package for FM benchmarking on AWS
+
+`FMBench` runs inference requests against endpoints that are either deployed through `FMBench` itself (as in the case of SageMaker) or are available either as a fully-managed endpoint (as in the case of Bedrock) or as bring your own endpoint. The metrics such as inference latency, transactions per-minute, error rates and cost per transactions are captured and presented in the form of a Markdown report containing explanatory text, tables and figures. The figures and tables in the report provide insights into what might be the best serving stack (instance type, inference container and configuration parameters) for a given Llama model for a given use-case.
+
+The following figure gives an example of the price performance numbers that include inference latency, transactions per-minute and concurrency level for running the `Llama2-13b` model on different instance types available on SageMaker using prompts for Q&A task created from the [`LongBench`](https://huggingface.co/datasets/THUDM/LongBench) dataset, these prompts are between 3000 to 3840 tokens in length. **_Note that the numbers are hidden in this figure but you would be able to see them when you run `FMBench` yourself_**.
+
+![`Llama2-13b` on different instance types ](./img/instances.png)
+
+The following table (also included in the report) provides information about the best available instance type for that experiment<sup>1</sup>.
+
+|Information	|Value	|
+|---	|---	|
+|experiment_name	|llama2-13b-inf2.24xlarge	|
+|payload_file	|payload_en_3000-3840.jsonl	|
+|instance_type	|ml.inf2.24xlarge	|
+|concurrency	|**	|
+|error_rate	|**	|
+|prompt_token_count_mean	|3394	|
+|prompt_token_throughput	|2400	|
+|completion_token_count_mean	|31	|
+|completion_token_throughput	|15	|
+|latency_mean	|**	|
+|latency_p50	|**	|
+|latency_p95	|**	|
+|latency_p99	|**	|
+|transactions_per_minute	|**	|
+|price_per_txn	|**	|
+
+<sup>1</sup> ** represent values hidden on purpose, these are available when you run the tool yourself.
+
+The report also includes latency Vs prompt size charts for different concurrency levels. As expected, inference latency increases as prompt size increases but what is interesting to note is that the increase is much more at higher concurrency levels (and this behavior varies with instance types).
+
+![Effect of prompt size on inference latency for different concurrency levels](./img/latency_vs_tokens.png)
+
+### How to get started with `FMBench`
+
+The following steps provide a Quick start guide for `FMBench`. For a more detailed DIY version, please see the [`FMBench Readme`](https://github.com/aws-samples/foundation-model-benchmarking-tool?tab=readme-ov-file#the-diy-version-with-gory-details).
```

Comment:
```suggestion
The following steps provide a [Quick start guide for `FMBench`](https://github.com/aws-samples/foundation-model-benchmarking-tool#quickstart). For a more detailed DIY version, please see the [`FMBench Readme`](https://github.com/aws-samples/foundation-model-benchmarking-tool?tab=readme-ov-file#the-diy-version-with-gory-details).
```

Reply:
Wonder DIY section is part of quickstart? or if someone does quick start would not need it?

Reply:
Maybe good to clarify if quick start handles the following steps ( step1?)